pkgs/tools/networking/uwimap/clang-fix.patch

```diff
@@ -0,0 +1,308 @@
+Only in b: ip6
+Only in b: osxok
```

Comment:
Nit: remove cruft.

Reply:
Thanks. I pushed an update PR without the cruft.